recipes/openmm-atmmetaforce-plugin/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+  skip: true  # [cuda_compiler_version in (undefined, 'None')]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - cmake
+    - swig
+
+  host:
+    - python
+    - openmm
+    - pthread-stubs
+    - lxml
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
+    
+  run:
+    - python
+    - openmm
```

```suggestion
    # openmm package is missing a run_export as of 9 June 2022
    - {{ pin_compatible('openmm') }}
```
Ensures that future API changes to openmm's C interface don't break your package.